practice/src/javascripts/sidebar.js

```diff
@@ -0,0 +1,13 @@
+// Identify the links in the sidebar and add/remove the 'active' class when clicked
+document.querySelector('.nav-list').addEventListener('click', function (event) {
+  // Check if the clicked element is a nav-link inside a nav-item
+  if (event.target.classList.contains('nav-link')) {
+    // Remove 'active' class from all nav-items
+    document
+      .querySelectorAll('.nav-item')
```

Comment:
Should use querySelectorAll('.nav-item.active') will minimize the number of elements traversed

Reply:
I have fixed it